### PR TITLE
Improve error messaging when role or env is missing, fixes #888

### LINKED ISSUE #888: Error: Argument cluster/[role]/[env] is not correct
Hi,

I got this error upon executing heron submit on aurora.
```
$ heron submit aurora --config-path ~/.heron/conf/ ~/.heron/examples/heron-examples.jar com.twitter.heron.examples.ExclamationTopology ExclamationTopology
ERROR: Argument cluster/[role]/[env] is not correct: role required but not provided
```

### PR DESCRIPTION
**Pull request by billonahill**

### CODE REVIEW

**Comment by nlu90** [comment]
👍

File: heron/cli/src/python/utils.py [hunk 1]
```diff
@@ -212,14 +212,14 @@ def parse_cluster_role_env(cluster_role_env, config_path):
       # if role is required but not provided, raise exception
       if len(parts) == 1:
         if (IS_ROLE_REQUIRED in cli_confs) and (cli_confs[IS_ROLE_REQUIRED] == True):
-          raise Exception("role required but not provided")
+          raise Exception("role required but not provided. See %s in %s" % (IS_ROLE_REQUIRED, CLIENT_YAML))
```
Comment (maosongfu) [inline heron/cli/src/python/utils.py]: Can u also print out the content of "parts"?

**Comment by billonahill** [comment]
@maosongfu done. I'm echoing back `cluster_role_env`, from which we can infer parts.

**Comment by maosongfu** [comment]
👍

**Pull request merged**